src/saving/SaveManagerGUI.cs

```diff
@@ -162,17 +167,21 @@ private void RefreshSaveCounts()
         getTotalSaveCountTask = new Task<(int Count, ulong DiskSpace)>(() => SaveHelper.CountSaves());
         getAutoSaveCountTask = new Task<(int Count, ulong DiskSpace)>(() => SaveHelper.CountSaves("auto_save"));
         getQuickSaveCountTask = new Task<(int Count, ulong DiskSpace)>(() => SaveHelper.CountSaves("quick_save"));
+        getOldBackupCountTask = new Task<(int Count, ulong DiskSpace)>(() => SaveHelper.CountOldBackupSaves());
```

Comment:
If you merge the method as i said you could have the ".backup" here as an argument , and change the CountSaves(nameStartsWith) argument name to "nameContains" inside the function itself.

Reply:
I but we might also have time implement some logic to check versions too, we don't want to go deleting backups with the same version as the game

Reply:
> But we might also have time implement some logic to check versions too, we don't want to go deleting backups with the same version as the game

I would not worry about that at all. If someone has played a newer version of the game and then goes back is a very convoluted use case. Anyway if they had cleaned the old files while in the new version, their backup saves would be already gone. Too much complexity for no gain in my opinion in trying to handle that.